internal/server/templates/index.html

```diff
@@ -13,83 +45,71 @@
 		label {
   			margin: 0.2rem 0;
 		}
-		.data-iframe {
-  			position: relative;
-  			top: 16px; 
-  			width: 98%;
-  			min-width: 768px;
-  			height: calc(100vh - 108px); 
-		}
+		
 	</style>
 </head>
 <body>
-<p>
-<form id="urlForm" action="/extract" method="POST" name="scrape" target="data-iframe">
+<div class="page-container">
+<div class="controls-container">
+<form x-data="formHandler" @submit.prevent="handleSubmit">
 <label for="url">Enter a URL:</label>
 <input type="submit" value="Hit It">
 <input type="url" name="url" id="url" value="https://" size="96" maxlength="200" pattern="https?://.*" required title="URL">
-<select title="URL Type" id="actionSelect" onchange="updateFormAction()">
+<select title="URL Type" x-model="formAction">
 	<option value="/extract">Page</option>
 	<option value="/extract/headless">Headless</option>
 	<option value="/feed">Feed</option>
 </select>
-<input type="hidden" name="token" value="{{AuthToken}}">
+<!-- <label for="token">Token:</label> -->
+<input type="hidden" name="token" size="101" maxlength="255" value="{{AuthToken}}" placeholder="Enter your access token">
 <input type="hidden" name="pp" value="1">
 </form>
-</p>
-<script>
-function updateFormAction() {
-    var selected = document.getElementById("actionSelect").value;
-    document.getElementById("urlForm").action = selected;
-}
-</script>
+</div><!-- controls-container -->
+<div class="response-container">
+	<pre x-ref="responseContent" id="responseContent"></pre>
+</div> 
+</div><!-- page-container -->
 <script type="text/javascript">
-	document.addEventListener('DOMContentLoaded', function() {
-		const iframe = document.getElementById('data');
-		iframe.contentDocument.body.innerHTML = '';
-		const preElement = document.createElement('pre');
-		preElement.style.whiteSpace = 'pre-wrap';
-		preElement.style.width = '100%';
-		iframe.contentDocument.body.appendChild(preElement);
-		const form = document.getElementById('urlForm');
-		form.addEventListener('submit', async function(event) {
-			event.preventDefault();
-			const action = form.action;
-			const headers = new Headers();
-			const token = form.token.value;
-			if (token) {
-				headers.append('Authorization', `Bearer ${token}`);
+	function formHandler() {
+		return {
+			formAction: '/extract',
+			async handleSubmit(event) {
+				const form = event.target;
+				const headers = new Headers();
+				const token = form.token.value;
+				if (token) {
+					headers.append('Authorization', `Bearer ${token}`);
+				}
+				const formData = new FormData(form);
+				formData.delete('token');
```

Comment:
Question: Why does the `token` deleted from the `formData` here?

Reply:
Only because we don't need to submit it with the form, it needs to go as a header. In the next PR I intend pull that out of the form entirely.